docs/ds4.browser.json

```diff
@@ -3,7 +3,6 @@
         "../src/less/less/ds4/variables.less",
         "../src/less/less/ds4/mixins.less",
         "src/less/ds4/index.less",
-        "./src/js/base/util.js",
```

Comment:
Where is the new polyfill being included in the bundle?

Reply:
Oh! Looks like I may have missed that. Duh! Thanks Sean.

Reply:
Although oddly it was working just fine as is in IE11. Still, it should be added to the browser.json.

Reply:
Oh, it was picking it up from nested dependency within makeup. Still, will add.